spec/controllers/state_file/questions/nj_year_of_death_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe StateFile::Questions::NjYearOfDeathController do
+  let(:intake) { create :state_file_nj_intake }
+
+  before { sign_in intake }
+
+  describe ".show?" do
+    context "when filer is qualifying widow/er" do
+      before do
+        allow(intake).to receive(:filing_status_qw?).and_return(true)
+      end
+
+      it "shows" do
+        expect(described_class).to be_show(intake)
+      end
+    end
+
+    context "when filer is not qss" do
+      before do
+        allow(intake).to receive(:filing_status_qw?).and_return(false)
+      end
+
+      it "does not show" do
+        expect(described_class).not_to be_show(intake)
+      end
+    end
+  end
+
+  describe "#update" do
+    let(:form_params) do
+      {
+        state_file_nj_year_of_death_form: {
+          spouse_death_year: 2023
```

Comment:
[dust] will eventually have to update this test when 2023 is no longer a valid choice... but will be quick and easy and in two years from now